controlplane/test/api-keys.test.ts

```diff
@@ -57,6 +58,10 @@ describe('API Keys', (ctx) => {
 
     const platformWebhooks = new MockPlatformWebhookService();
 
+    const prometheus = new PrometheusClient({
```

Comment:
How will it behave in tests when Prometheus is not running? We should either provide a mock implementation or conditionally disable it when we don't require prometheus in tests.

Reply:
Add mock client